saleor/graphql/core/mutations.py

```diff
@@ -609,6 +594,29 @@ def check_channel_permissions(
                 message="You don't have access to some objects' channel."
             )
 
+    @classmethod
+    def create_metadata_from_graphql_input(
+        cls, metadata_list: list[MetadataInput] | None, *, error_field_name: str
+    ) -> MetadataItemCollection:
+        """Wrap the creation of metadata and map the error to graphql error.
+
+        In case of metadata - we need to pass field name, because it can be nested in the other path than "metadata" or "privateMetadata"
```

Comment:
```suggestion
        In case of metadata - we need to pass `error_field_name`, because it can be nested in the other path than "metadata" or "privateMetadata"
```
It would be more meaningful to use exact name of the parameter that you're writing about